<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ws_gas_flow_predict_adj_dlg</class>
 <widget class="QWidget" name="ws_gas_flow_predict_adj_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>邻近层相对瓦斯涌出量q2计算</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../usecad.qrc">
    <normaloff>:/images/cbm.ico</normaloff>:/images/cbm.ico</iconset>
  </property>
  <widget class="QPushButton" name="add_adj_layer">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_adj_layer">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QComboBox" name="adj_layer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>邻近层列表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="thick">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>煤层厚度</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="n_cz">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>位于冒落带内</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>瓦斯排放率η</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gas_eta">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="eta_cacl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>计 算</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="h">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>岩层破坏范围hp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hp">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>与开采层垂直距离h</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gas_W0">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>煤层残存瓦斯含量Wc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gas_Wc2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>煤层原始瓦斯含量W0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保 存</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../usecad.qrc"/>
 </resources>
 <connections/>
</ui>
